src/test/java/co/rsk/federate/FederationProviderFromFederatorSupportTest.java

```diff
@@ -50,28 +55,27 @@ class FederationProviderFromFederatorSupportTest {
     private static final Script HARDCODED_TESTNET_FED_REDEEM_SCRIPT = new Script(
         Hex.decode("6453210208f40073a9e43b3e9103acec79767a6de9b0409749884e989960fee578012fce210225e892391625854128c5c4ea4340de0c2a70570f33db53426fc9c746597a03f42102afc230c2d355b1a577682b07bc2646041b5d0177af0f98395a46018da699b6da210344a3c38cd59afcba3edcebe143e025574594b001700dec41e59409bdbd0f2a0921039a060badbeb24bee49eb2063f616c0f0f0765d4ca646b20a88ce828f259fcdb955670300cd50b27552210216c23b2ea8e4f11c3f9e22711addb1d16a93964796913830856b568cc3ea21d3210275562901dd8faae20de0a4166362a4f82188db77dbed4ca887422ea1ec185f1421034db69f2112f4fb1bb6141bf6e2bd6631f0484d0bd95b16767902c9fe219d4a6f5368ae")
     );
+    private static final List<FederationMember> federationMembersFromPks = getFederationMembersFromPks(1, 1000, 2000, 3000, 4000, 5000);
+    private static final FederationConstants federationConstants = FederationTestNetConstants.getInstance();
+    private static final NetworkParameters testnetParams = NetworkParameters.fromID(NetworkParameters.ID_TESTNET);
+    private static final Instant creationTime = Instant.ofEpochSecond(5);
+    private static ActivationConfig.ForBlock configMock;
 
     private FederatorSupport federatorSupportMock;
     private FederationProvider federationProvider;
-    private FederationConstants federationConstants;
-    private NetworkParameters testnetParams;
-    private Instant creationTime;
 
     @BeforeEach
     void createProvider() {
-        federationConstants = FederationTestNetConstants.getInstance();
+        configMock = mock(ActivationConfig.ForBlock.class);
```

Comment:
```suggestion
        activations = mock(ActivationConfig.ForBlock.class);
```

We usually call it just activations. We tend not to add mock suffixes.

Reply:
done!